pkg/cloud/tags/tags.go

```diff
@@ -115,7 +115,11 @@ func WithEC2(ec2client ec2iface.EC2API) BuilderOption {
 			}
 
 			_, err := ec2client.CreateTags(createTagsInput)
-			return errors.Wrapf(err, "failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			if err != nil {
+				return errors.Wrapf(err, "failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			}
+			errmsg := fmt.Sprintf("failed to tag resource %q in cluster %q", params.ResourceID, params.ClusterName)
+			return errors.Errorf(errmsg)
```

Comment:
We should return nil when err == nil, meaning CreateTags are successful. This is why multiple tests are failing.

Reply:
Thanks @pydctw I reverted it back to the original way(since wrapf will return nil if err is nil) , the failing tests failed once but passed after squash